onnxruntime/test/testdata/transform/cse/cse1.onnx

```diff
@@ -0,0 +1,29 @@
+makalini:�
+
```

Comment:
Is it possible to add files that generate these onnx models so they're easily updated in the future? e.g. a lot of recent additions to testdata have .py files to create the model.